### PR TITLE
G1 and Fountains

### PR DESCRIPTION
**Pull request by Tonkatunk**
All the periods were changed into NA periods(.).  Translated fountain effects and G1 related stuff.   Black orb and seal.

When translated, sealstone.3 says "too weak to open the gate."  Should this translate to "can't open gate during the week?"

### CODE REVIEW

**Comment by domirade** [comment]
I don't see the point of changing periods in sentences that are otherwise untranslated from JP... it shouldn't hurt, but it seems unnecessary.

Also, you've introduced a new file `npc Tonka.japan.txt` that probably should not have been committed.

File: puzzle.japan.txt [hunk 1]
```diff
@@ -78,121 +78,121 @@ element.keychest_skeleton.5	single(smallskeletonarmorb:
...
+prop.red_fountain.56	が出た.
+prop.red_fountain.57	Red fountain gave {0}.
+prop.red_fountain.58	Red fountain effect faded.
+prop.red_fountain.59	Red fountain effect faded.
+prop.red_fountain.60	Red fountain gave {0}.
+prop.red_fountain.61	Red fountain effect faded.
+prop.red_fountain.62	Red fountain effect faded.
+prop.tirnanog_sealstone.1	Player touches the seal of Tir Na Nog.
+prop.tirnanog_sealstone.2	I am not able to break this seal.
+prop.tirnanog_sealstone.3	クリステルの話によると、\n封印の力が弱くなるサーオィンにならないと封印を解くことはできないそうだ.
```
Comment (domirade) [inline puzzle.japan.txt:183]: "According to Kristell, the seal cannot be broken until Samhain, when the power of the seal becomes weaker."

File: puzzle.japan.txt [hunk 2]
```diff
@@ -78,121 +78,121 @@ element.keychest_skeleton.5	single(smallskeletonarmorb:
...
+prop.tirnanog_sealstone.4	弱すぎて、門が開かないようだ.
```
Comment (domirade) [inline puzzle.japan.txt:184]: "It feels like I'm too weak to break the seal right now."

**Comment by Tonkatunk** [comment]
> I don't see the point of changing periods in sentences that are otherwise untranslated from JP... it shouldn't hurt, but it seems unnecessary.

Replace all function, was easier that way.  Similarly 

> Also, you've introduced a new file `npc Tonka.japan.txt` that probably should not have been committed.

Should be removed, thanks for pointing that out.

**Pull request merged**